src/index.ts

```diff
@@ -1077,7 +1085,7 @@ export class ValidationService {
 
         for (let i = 0; i < inputs.length; i++) {
             let input = inputs[i];
-            if (remove) {
+            if (remove || input.disabled) {
```

Comment:
Hmm… I'm not thrilled with needing to depend on `watch` to validate an input that has been un-`disabled`. That would be an unexpected deviation from jQuery Validation.

I think instead we want it to keep it wired up but make sure validation of a disabled field never fails and won't show a message or summary entry.